docs/man/git-lfs-clean.1.ronn

```diff
@@ -18,3 +18,4 @@ Git attributes.
 git-lfs-init(1), git-lfs-push(1), gitattributes(5).
 
 Part of the git-lfs(1) suite.
+
```

Comment:
I added some white space to the end of some man files to make them display better in the terminal